src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.cs

```diff
@@ -33,6 +33,8 @@ internal partial class ExplicitInterfaceMemberCompletionProvider : CommonComplet
                     SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers |
                     SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
 
+        private static ImmutableArray<MethodKind> s_invalidCodeCompletionMethodKind = ImmutableArray.Create(new MethodKind[] { MethodKind.PropertyGet, MethodKind.PropertySet, MethodKind.EventAdd, MethodKind.EventRemove });
```

Comment:
ImmutableArray.Create takes a params array -- you can drop the new MethodKind bit inside of there.

Reply:
Thanks for the feedback. I ended up removing this since there is an IsAccessor extension that makes more sense to use.